supervisord.conf

```diff
@@ -3,6 +3,8 @@ nodaemon = true
 
 [program:nginx]
 command = /nginx.sh
+# scrub potentially sensitive environment variables we don't need
+environment = AWS_ACCESS_KEY_ID="",AWS_SECRET_ACCESS_KEY="",DM_ADMIN_USER_IPS="",DM_ANTIVIRUS_API_AUTH_TOKENS="",DM_ANTIVIRUS_API_CALLBACK_AUTH_TOKENS="",DM_API_AUTH_TOKENS="",DM_API_CALLBACK_AUTH_TOKENS="",DM_APP_AUTH="",DM_DATA_API_AUTH_TOKEN="",DM_DEV_USER_IPS="",DM_DNB_API_PASSWORD="",DM_MAILCHIMP_API_KEY="",DM_MANDRILL_API_KEY="",DM_NOTIFY_API_KEY="",DM_SEARCH_API_AUTH_TOKEN="",DM_SEARCH_API_AUTH_TOKENS="",DM_USER_IPS="",SECRET_KEY="",SHARED_EMAIL_KEY=""
```

Comment:
I don't suppose we can wrap these lines?

Reply:
Hooray it looks like it should be possible: https://docs.python.org/3/library/configparser.html#supported-ini-file-structure